maquetado/scss/styles.scss

```
@@ -39,7 +39,7 @@
 
 .redirect-arrow {
   font-size: 20px;
-  margin-left: 100px;
+  margin-left: 0 0 60px 100px;
```

You are using a shorthand. This should be 

```suggestion
  margin: 0 0 60px 100px;
```